PWGJE/Tasks/hffragmentationfunction.cxx

```diff
@@ -68,14 +84,68 @@ DECLARE_SOA_TABLE(JetDistanceTable, "AOD", "JETDISTTABLE",
                   DistanceSpace::HfPhi,
                   DistanceSpace::HfMass,
                   DistanceSpace::HfY);
+namespace MCPDistanceTable
+{
+DECLARE_SOA_COLUMN(JetHfDist, jethfdist, float);
+DECLARE_SOA_COLUMN(JetPt, jetpt, float);
+DECLARE_SOA_COLUMN(JetEta, jeteta, float);
+DECLARE_SOA_COLUMN(JetPhi, jetphi, float);
+DECLARE_SOA_COLUMN(HfPt, hfpt, float);
+DECLARE_SOA_COLUMN(HfEta, hfeta, float);
+DECLARE_SOA_COLUMN(HfPhi, hfphi, float);
+DECLARE_SOA_COLUMN(HfY, hfy, float);
+DECLARE_SOA_COLUMN(HfMatch, hfmatch, bool);
+} // namespace MCPDistanceTable
+DECLARE_SOA_TABLE(MCPJetDistanceTable, "AOD", "MCPJETDISTTABLE",
+                  MCPDistanceTable::JetHfDist,
+                  MCPDistanceTable::JetPt,
+                  MCPDistanceTable::JetEta,
+                  MCPDistanceTable::JetPhi,
+                  MCPDistanceTable::HfPt,
+                  MCPDistanceTable::HfEta,
+                  MCPDistanceTable::HfPhi,
+                  MCPDistanceTable::HfY,
+                  MCPDistanceTable::HfMatch);
+namespace MCDDistanceTable
+{
+DECLARE_SOA_COLUMN(JetHfDist, jethfdist, float);
+DECLARE_SOA_COLUMN(JetPt, jetpt, float);
+DECLARE_SOA_COLUMN(JetEta, jeteta, float);
+DECLARE_SOA_COLUMN(JetPhi, jetphi, float);
+DECLARE_SOA_COLUMN(HfPt, hfpt, float);
+DECLARE_SOA_COLUMN(HfEta, hfeta, float);
+DECLARE_SOA_COLUMN(HfPhi, hfphi, float);
+DECLARE_SOA_COLUMN(HfMass, hfmass, float);
+DECLARE_SOA_COLUMN(HfY, hfy, float);
+DECLARE_SOA_COLUMN(HfMatch, hfmatch, bool);
```

Comment:
There is no need to duplicate these column declarations.

Reply:
You are right, I just removed the unnecessary ones